bootloader/src/pyi_global.h

```diff
@@ -79,6 +79,99 @@ typedef int bool;
     #include <limits.h>  /* PATH_MAX */
 #endif
 
+/*
+ * These macros were moved here from pyi_python.h to illustrate
+ * that they can be used in other places too.
+ */
+/*
+ * These macros used to define variables to hold dynamically accessed entry
+ * points. These are declared 'extern' in the header, and defined fully later.
+ */
```

Comment:
I was also thinking about joining the two comments together, but I decided to leave the two separate, since the second comment is the original comment from `pyi_python.h` and the first one is just a hint that this part has been moved.